File: src/Easy/kidsWithMostCandies/kidsWithMostCandies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        max_candies = max(candies)
        # res = [True if kid + extraCandies >= max_candies else False for kid in candies]
        return [True if kid + extraCandies >= max_candies else False for kid in candies]

File: src/Easy/kidsWithMostCandies/kidsWithMostCandies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */

function kidsWithMostCandies(candies, extraCandies) {
  const max = Math.max(...candies);
  const res = [];
  for (let i = 0; i < candies.length; i++) {
    const kid = candies[i];
    if (kid + extraCandies >= max) {
      res.push(true);
    } else {
      res.push(false);
    }
  }
  return res;
}

module.exports = kidsWithMostCandies;
